skills/trilogy-ds/rules/composition.md

```
@@ -0,0 +1,330 @@
+
+// DON'T ❌ — Imports profonds
+import { Section } from '@trilogy-ds/react/components/section'
+```
+
+---
+
+## Fallback — API Reference
+
+Pour les props non documentées ici, consulter la référence complète :
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)